controllers/argocd_controller.go

```diff
@@ -117,6 +118,11 @@ func (r *ReconcileArgoCDRoute) Reconcile(ctx context.Context, request reconcile.
 	reqLogger := logs.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling ArgoCD Route")
 
+	if !util.IsConsoleAPIFound() {
+		reqLogger.Info("Skip argocd route reconcile: OpenShift Console API not found")
+		return reconcile.Result{}, nil
+	}
+
```

Comment:
I think the route should be reconciled regardless of the availability of ConsoleLink API?

Reply:
This reconcile method doesn't reconcile the route, it just updates the consolelink with the existing route, or deletes the console link if the route doesn't exist.

Reply:
Ah. Now as you say it, and looking further down the code... :) 

Then maybe only the log message (and also the existing ones!) are misleading. Because this method doesn't reconcile a route, but a ConsoleLink.